core/src/main/java/org/geysermc/geyser/text/MinecraftLocale.java

```diff
@@ -80,6 +82,11 @@ public static void downloadAndLoadLocale(String locale) {
             locale = "no_no";
         }
 
+        if (isLocaleLoaded(locale)) {
```

Comment:
This will probably fail for nb_no; as isLocaleLoaded checks the locale map that holds the Bedrock locale "codes", but this was replaced with the Java locale code. In other words; this should be moved to before we replace `nb_no` with `no_no` .